src/sync/models/centralizedOracle.js

```diff
@@ -14,26 +14,27 @@ class CentralizedOracle {
   }
 
   decode() {
-    let nameHex = _.reduce(this.rawLog['_name'], (hexStr, value) => {
-      return hexStr += value;
-    }, '');
-    this.name = Utils.toUtf8(nameHex);
-
-    let intermedia = _.map(this.rawLog['_resultNames'], (item) => Utils.toUtf8(item));
-    this.resultNames = _.filter(intermedia, item => !!item);
-
+    this.version = this.rawLog['_version'].toNumber();
     this.contractAddress = this.rawLog['_contractAddress'];
     this.oracle = this.rawLog['_oracle'];
     this.eventAddress = this.rawLog['_eventAddress'];
     this.numOfResults = this.rawLog['_numOfResults'].toNumber();
+    this.bettingStartBlock = this.rawLog['_bettingStartBlock'].toNumber();
     this.bettingEndBlock = this.rawLog['_bettingEndBlock'].toNumber();
+    this.resultSettingStartBlock = this.rawLog['_resultSettingStartBlock'].toNumber();
     this.resultSettingEndBlock = this.rawLog['_resultSettingEndBlock'].toNumber();
     this.consensusThreshold = this.rawLog['_consensusThreshold'].toJSON();
   }
 
+  setTopicInfo(topic) {
```

Comment:
this should be in synchroniser file as model is direct translation of blockchain event

Reply:
done